src/Footer.js

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { styled, alpha } from '@mui/material/styles';
+
+const FooterContainer = styled('div')(({ theme }) => ({
+    position: 'fixed',
+    left : 0,
+    bottom : 0,
+    color : 'white',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderRadius: theme.shape.borderRadius,
+    backgroundColor: alpha(theme.palette.common.black,1),
+    // '&:hover': {
+    //   backgroundColor: alpha(theme.palette.common.black, 1),
+    // },
+    width: '100%',
+    height : '90px',
+  }));
+
+const UseMapSpan = styled('span')(({theme}) => ({
+  postion: 'absolute',
+  left: '10px',
+  
+  // display: 'flex',
+  // justifyContent: 'center',
+  padding: 5,
+}));
+
+const UseMapLink = styled('a')(({theme}) => ({
+  '&:link': {
+    color: theme.palette.common.white,
+  },
+  '&:hover': {
+    color: theme.palette.common.red,
+  }
+}));
+
+function Footer()
+{
+    return (
+        <FooterContainer>
+          <img height="75" src={process.env.PUBLIC_URL + "/elelogo_transp.png"} />
```

Comment:
The indentation here looks slightly wrong (img should be aligned with UseMapSpan). Do you have a formatter set up with VSCode?